src/thumbnail/external-thumbnails-handler.js

```diff
@@ -241,10 +241,10 @@ class ExternalThumbnailsHandler extends FakeEventTarget {
       size = {width, height};
       isValidThumbnailVTTFormat = [x, y, width, height, imgUrl].every(option => option !== undefined);
     } else {
-      imgUrl = text;
+      imgUrl = text[0] === '/' ? text.substring(1) : text;
```

Comment:
I think it always should be relative. unless the text is starting with 'http' or '//'. @OrenMe wdyt?

Reply:
so if we have a relative url in the vtt file we set the image url as `/myImagePath/1.png` and then it get the hosting page domain instead of the vtt file?
I think that what you say @yairans is more safe, check if thumb image url starts with http|s and if so take it as is, otherwise treat it as relative

Reply:
no, it will always get the vtt file domain (and path) unless it is absolute path

Reply:
@JonathanTGold now you consider it as relative only if it's starting with '/'. I think you should consider it relative even it's not starting with '/', provided it's not starting with 'http' or '//'. e.g.
'/thumbnail.png' = 'thumbnail.png' = relative. 
'http//:...thumbnail.png' = '//....thumbnail.png' = absolute.

Reply:
updated, now any case beside '/' in the begining of the string (absulut path) would remain as it is, otherwise would be resolved to relative pathe reating to the vtt file location